quarkchain/cluster/tests/test_filter.py

```diff
@@ -123,3 +143,28 @@ def test_get_logs(self):
         f = self.filter_gen_with_criteria(criteria, addresses)
         logs = f._get_logs([self.hit_block])
         self.assertListEqual([self.log], logs)
+
+    def test_get_block_candidates_height_ascending(self):
+        criteria = []
+        addresses = []
+        f = self.filter_gen_with_criteria(criteria, addresses)
+        blocks = f._get_block_candidates()
+
+        height = 1
+        for block in blocks:
+            self.assertEqual(block.header.height, height)
+            height += 1
+
+        f = self.filter_gen_with_criteria(criteria, addresses, option="block_list")
+        blocks = f._get_block_candidates()
+        height = 2
+        for block in blocks:
```

Comment:
similar `self.assertListEqual([b.header.height for b in blocks], list(range(2, 11))`